clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -809,7 +809,7 @@ public void run() {
 
                 // Handle new calls and metadata update requests.
                 prevMetadataVersion = checkMetadataReady(prevMetadataVersion);
-                long pollTimeout = 1200000;
+                long pollTimeout = 100;
```

Comment:
Removed this hack as it seems to be fixed by the @cmccabe's timeout fixes PR that I just merged.